Kitodo/src/main/java/org/goobi/io/SafeFile.java

```diff
@@ -315,4 +315,8 @@ public URI toURI() {
     public URL toURL() throws MalformedURLException {
         return delegate.toURL();
     }
+
+    public File getDelegate() {
```

Comment:
I will not change the method signature back, as it will be followed by a huge trace of using the SafeFile class in the FileService, which I cleaned up properly.
The SafeFile class will be removed in the next PullRequest